lib/srv/desktop/rdp/rdpclient/Cargo.toml

```diff
@@ -20,7 +20,7 @@ num-traits = "0.2.16"
 rand = { version = "0.8.5", features = ["getrandom"] }
 rand_chacha = "0.3.1"
 rsa = "0.9.2"
-rdp-rs = { git = "https://github.com/gravitational/rdp-rs", rev = "75eb6a30b83e7152ee6213964b5ac6e783304840" }
+rdp-rs = { git = "https://github.com/gravitational/rdp-rs", rev = "4c09a075b3a51fd2aa0e12be64c8096c332f042e" }
```

Comment:
Reminder not to merge this until the rdp-rs PR has been merged and this ref is updated (it points at my def branch curently)

Reply:
Merged: https://github.com/gravitational/rdp-rs/pull/34